openziti.yml

```diff
@@ -0,0 +1,93 @@
+services:
+  openziti:
+    container_name: edgex-openziti
+    image: openziti/ziti-cli:0.33.1
+    restart: unless-stopped
+    networks:
+      edgex-network:
+        aliases:
+          - openziti
+          - ziti-controller
+          - ziti-edge-controller
+          - ziti-router
+    entrypoint:
+    - bash
+    - -euc
+    - |
+      ZITI_CMD+=" --ctrl-address ${EXTERNAL_DNS:-openziti}"\
+      " --ctrl-port ${ZITI_CTRL_EDGE_ADVERTISED_PORT:-1280}"\
+      " --router-address ${EXTERNAL_DNS:-openziti}"\
+      " --router-port ${ZITI_ROUTER_PORT:-3022}"\
+      " --password ${ZITI_PWD:-admin}"
+      exec ziti "$${@}" $${ZITI_CMD}
```

Comment:
Put this at the bottom of your entry point script.

Reply:
nevermind

Reply:
There are three containers in this compose file. One runs OpenZiti (this one). The other two will:
* chmod the named volume for writing
* initialize OpenZiti, after it's running

This is the "run OpenZiti" section and cannot be moved to after the "configure OpenZiti" section.

Reply:
Yep, figured that out... ;-)